app/views/wiki/_header.html.erb

```diff
@@ -70,22 +70,23 @@
 
       <% if current_user && current_user.role == "admin" %>
         <li>
-          <%= link_to url_for(action: :delete,id: @node.id),method: :delete, data: {confirm: I18n.t('wiki.show.are_you_sure_delete', path: @node.path)} do %>
+          <a rel="tooltip" title="Delete this wiki page." data-placement="top" %= link_to url_for(action: :delete,id: @node.id),method: :delete, data: {confirm: I18n.t('wiki.show.are_you_sure_delete', path: @node.path)} do %>
             <i class="fa fa-trash"></i>
             <span class="hidden-xs hidden-sm hidden-md hidden-lg"> <%= t('wiki.show.delete') %></span>
+          </a>
           <% end %>
         </li>
       <% end %>
 
       <li>
-        <a href="/talk/<%= @node.slug_from_path %>">
+        < a rel="tooltip" title="Practice in a realtime doc." data-placement="top" href="/talk/<%= @node.slug_from_path %>">
```

Comment:
Could you please remove the whitespace between `<` and `a`? That might fix the problem.

Reply:
Thanks! I have done the same ,but i feel still there is some error.